From e4af3b6f62d46e1c3299e161cd1669df2455779b Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Tue, 17 Dec 2024 05:38:26 +0800
Subject: [PATCH 308/314] AOSCOS: MIPS: remove duplicate defines of NR_syscalls

In "FROMLIST: MIPS: Add syscall auditing support", some duplicate
defines of NR_syscalls were added to arch/mips/include/asm/unistd.h,
which were already added by
1d7bf993e073 ("MIPS: ftrace: Add support for syscall tracepoints.").

Removing the duplicate defines of NR_syscalls.

Fixes: "FROMLIST: MIPS: Add syscall auditing support"
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/include/asm/unistd.h | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/arch/mips/include/asm/unistd.h b/arch/mips/include/asm/unistd.h
index 1c054e3096db..ba83d3fb0a84 100644
--- a/arch/mips/include/asm/unistd.h
+++ b/arch/mips/include/asm/unistd.h
@@ -64,14 +64,4 @@
 
 #endif /* !__ASSEMBLY__ */
 
-#ifdef CONFIG_MIPS32_N32
-#define NR_syscalls	(__NR_N32_Linux + __NR_N32_Linux_syscalls)
-#elif defined(CONFIG_64BIT)
-#define NR_syscalls	(__NR_64_Linux  + __NR_64_Linux_syscalls)
-#elif defined(CONFIG_32BIT)
-#define NR_syscalls	(__NR_O32_Linux + __NR_O32_Linux_syscalls)
-#else
-#error Must know ABIs in use to define NR_syscalls
-#endif
-
 #endif /* _ASM_UNISTD_H */
-- 
2.48.1

